frontend/src/components/DataViewDrawer.vue

```diff
@@ -47,9 +47,17 @@ const query = async (filters = {}) => {
 
   // Delete the 'Figure model (Hand-drawn)' key
   delete counts['Figure model (Hand-drawn)']
+  let total = 0
+  const cleanedCounts = {}
 
-  modelTypeCounts.value = counts
-  totalModels.value = Object.values(counts).reduce((acc, count) => acc + count, 0)
+  for (const [key, value] of Object.entries(counts)) {
+    const cleanedKey = key.replace(' model', '')
+    cleanedCounts[cleanedKey] = value
+    total += value
+  }
+
+  totalModels.value = total
+  modelTypeCounts.value = cleanedCounts
```

Comment:
because this is in the `<script setup>` it only executes once. So if you do any filtering, the "model" part of the text is shown again:

https://github.com/user-attachments/assets/a68c56cf-0615-411b-babc-895e3527b8e2